airbyte-webapp/src/pages/SourcesPage/pages/SourceItemPage/components/SourceSettings.tsx

```diff
@@ -58,7 +58,7 @@ const SourceSettings: React.FC<IProps> = ({
 
       setSaved(true);
     } catch (e) {
-      const errorStatusMessage = createFormErrorMessage(e.status);
+      const errorStatusMessage = e.message || createFormErrorMessage(e.status);
```

Comment:
would it make sense to put pass `e.message` to `createFormErrorMessage` and change its method signature to look like this `createFormErrorMessage(status, message)` and have that handle the logic so that you don't need to have this or statement everywhere?